app/frontend/src/common/mapbox/layers.js

```diff
@@ -278,12 +278,16 @@ export function wellsBaseAndArtesianLayer (options = {}) {
   const styles = defaultsDeep(options.styles, {
     'circle-color': [
       'case',
+      ['==', ['get', 'well_status'], 'CLOSURE'], '#302d57',
+      ['==', ['get', 'well_status'], 'ABANDONED'], '#302d57',
       ['to-boolean', ['get', 'artesian']], '#1099FE',
       '#0162FE'
     ],
     'circle-radius': 3,
     'circle-stroke-color': [
       'case',
+      ['==', ['get', 'well_status'], 'CLOSURE'], 'transparent',
+      ['==', ['get', 'well_status'], 'ABANDONED'], 'transparent',
       ['to-boolean', ['get', 'artesian']], '#EE14CA',
```

Comment:
When you're presenting the changes to the SO, make sure you highlight everything listed in the PR (sometimes the comments don't get read) and specifically this one. Given my understanding of how artesian conditions are used, they might want wells that have them to still be highlighted even when closed. I'm not saying undo the change right now, but double check with the SO.

Reply:
Ack, I will do that now, thank you!